composer.json

```diff
@@ -21,14 +21,15 @@
         "php-http/discovery": "^1.0",
         "php-http/httplug": "^1.1 || ^2.0",
         "php-http/message": "^1.6",
-        "php-http/message-factory": "^1.0",
         "php-http/promise": "^1.0",
+        "psr/http-factory": "^1.0",
         "psr/http-message": "^1.0",
         "ramsey/uuid": "^3.7 || ^4.0"
     },
     "require-dev": {
         "ergebnis/composer-normalize": "^2.4",
-        "lmc/coding-standard": "^1.3 || ^2.0",
+        "http-interop/http-factory-guzzle": "^1.2",
+        "lmc/coding-standard": "^1.3 || ^2.0 || ^3.0",
```

Comment:
Its not possible to make it compatible to all these version, because there are incompatible changes in 3.0 (especially config format changed to PHP).

I think https://github.com/lmc-eu/matej-client-php/pull/135/files should be resurrected and finished first. Are you able to took over the PR #135 and finish it?

Reply:
Yes, it's ok to change it in #135, I don't need it now. It's because of PHP8.3 need it.